src/onemkl/onemkl_potrf.cc

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) 2017-2020, University of Tennessee. All rights reserved.
+// SPDX-License-Identifier: BSD-3-Clause
+// This program is free software: you can redistribute it and/or modify it under
+// the terms of the BSD 3-Clause license. See the accompanying LICENSE file.
+
+#include "lapack/defines.h"
+
+#if defined(LAPACK_HAVE_ONEMKL)
+
+#include "onemkl_common.hh"
+
+//==============================================================================
+// todo: put into BLAS++ header somewhere.
+
+namespace blas {
+namespace device {
```

Comment:
This changed from `device` to `internal` in BLAS++. I'm fixing the CUDA and ROCm files.

Reply:
I fixed that in the recent commit.
https://github.com/icl-utk-edu/lapackpp/pull/17/commits/a74fd096da6fb51d055b7c6b1eae443ba6e65dd3